system_test/aest_sync_SUITE.erl

```diff
@@ -753,10 +753,6 @@ node_mined_retries(Nodes) ->
         end
     end, 0, Nodes).
 
-ping_interval() ->
-    aeu_env:user_config_or_env([<<"sync">>, <<"ping_interval">>],
-                               aecore, ping_interval, 120000).
```

Comment:
This was not a faithful representation of the value in the node.

Reply:
That's correct. May I ask you to use the aest_nodes:get_node_config/2 function? This gets the config that is on the node via the state stored when setting up that node

Reply:
Addressed by force-pushing. This is meant to be pure refactoring, though by running locally again I got a failure as it seems that when domain name cannot be resolved the nodes seem to stop attempting to contact each other for long. Checking.

Reply:
> ... by running locally again I got a failure as it seems that when domain name cannot be resolved the nodes seem to stop attempting to contact each other for long. Checking.

Addressed in new commit 43612989

Reply:
For the record: I did not manage to reproduce locally that failure with default hostname resolution backoff times, though I verified that with the new config in the sync log the node manages connect to the peers in the just-connected network much quicker.